examples/plot_raster_stack_crop.py

```diff
@@ -134,9 +132,7 @@
 # function. Do you notice any extreme values that may be impacting the stretch
 # of the image?
 
-ep.hist(array, title=["Band 1",
-                      "Band 2",
-                      "Band 3"])
+ep.hist(array, title=["Band 1", "Band 2", "Band 3"])
 plt.show()
```

Comment:
i can't remember. do we need `plt.show()` for hist ? it's been a while :)

Reply:
Just tested it without `plt.show()`, it came up with the matplotlib image stuff sadly